akka-testkit/src/main/scala/akka/testkit/TestActors.scala

```diff
@@ -19,6 +19,18 @@ object TestActors {
     }
   }
 
+  /**
+   * ForwardActor forwards all messages as-is to specified ActorRef.
+   *
+   * @param ref target ActorRef to forward messages to
+   */
+  class ForwardActor(ref: ActorRef) extends Actor {
+    override def receive = {
+      case message ⇒ ref forward message
+    }
+  }
+
   val echoActorProps = Props[EchoActor]()
+  val forwardActorProps: (ActorRef ⇒ Props) = Props(classOf[ForwardActor], _)
```

Comment:
Also, the val version will allocate a Function1, while the def version will not, unless it is used in a closure. Not that it matters, but we were talking about OCD :)